src/gsi/rapidrefresh_cldsurf_mod.f90

```diff
@@ -181,6 +185,18 @@ module rapidrefresh_cldsurf_mod
 !                          = 2(clean Qg as in 1, and adjustment to the retrieved Qr/Qs/Qnr throughout the whole profile)
 !                          = 3(similar to 2, but adjustment to Qr/Qs/Qnr only below maximum reflectivity level
 !                           and where the dbz_obs is missing);
+!      corp_howv      - namelist real, static BE of howv (standard error deviation)
+!      hwllp_howv     - namelist real, static BE de-correlation length scale of howv
+!      i_howv_in_3dda - integer, control the analysis of howv in 3D analysis (either var or hybrid)
+!                          = 0 (howv-off: default) : no analysis of howv in 3D analysis.
+!                          = 1 (howv-on) : if variable name "howv" is found in anavinfo,
+!                                          set it to be 1 to turn on analysis of howv;
```

Comment:
@guoqing-noaa 
Based on current testing results, if howv is added in anavinfo (esp. in the ctrl-vec and stat-vec), howv would be added in the cost function and have impact on the minimisation. So simply using i_howv_3dda could not control the analysis of howv.  Anyway, using a namelist option to control the analysis of howv might be useful in the future version, it could be revisited later.
Thank you!